R/widget-get-value.R

```diff
@@ -35,6 +35,10 @@ widget_getValueFuncs <- list(
 
   textOutput = function(self, private) {
     private$element$getText()
+  },
+
+  plotOutput = function(self, private) {
+    private$element$findElement("img")$getAttribute("src")
```

Comment:
@wch This is the part that you were saying should already exist.  I was trying to retrieve values using `app$getValue("distPlot", iotype = "output")` and it would create an error.  Returning the full image object that you described earlier would be **much** better.